debian/pghoard.postinst

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
+
+#DEBHELPER#
+
+if [ "$1" = triggered ]; then
```

Comment:
postinst is called with "configure" as the first arg.

https://www.debian.org/doc/debian-policy/ch-maintainerscripts.html